Source/Scene/Cesium3DTile.js

```diff
@@ -305,6 +307,13 @@ define([
          * @private
          */
         this.visibilityPlaneMask = true;
+        
+        /**
+         * Flag to mark children visibility
+         *
+         * @private
+         */
+        this.childrenVisibility = 1;
```

Comment:
Instead of assigning to `1`, `ChildrenVisibilityFlags` should be in its own file (it can be `@private`) and declared like other Cesium enums.  For example, see [Cesium3DTileContentState.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Cesium3DTileContentState.js).